packages/app/cypress/e2e/cypress-in-cypress-e2e.cy.ts

```diff
@@ -58,6 +81,26 @@ describe('Cypress In Cypress', { viewportWidth: 1200 }, () => {
     cy.contains('dom-content.spec').click()
     cy.get('[data-model-state="passed"]').should('contain', 'renders the test content')
 
+    let styleBuffer = ''
+
+    cy.get('link[rel=stylesheet][href]').each(($el) => {
```

Comment:
I wonder if we want to make this a custom command, or at least put in it a function explaining what it does (I have no idea why this is necessary).

Reply:
Oh yes - I was just trying some stuff out tonight after talking with Tyler. Let me convert this to draft to avoid confusion. Percy snapshots from cypress-in-cypress tests don't have any styles from the Vue app. I'm hoping that something like this, to get it working, will reveal a bit more about what a permanent solution would look like. Probably it can be solved in a way that doesn't need this kind of thing.

Reply:
Removed now!